templates/generate.html.j2

```diff
@@ -18,10 +18,10 @@
 
     <p>This page provides instructions for generating a new SSH keypair for use with S3DF. Specifically, we present shell commands that you should paste into your local (eg laptop) terminal that will create a SSH private and public keypair. You will then be able to utilize this keypair to SSH into S3DF without having to always enter a username, password and 2factor (duo).</p>
     
-    <p>Our implementation also considers the use of time limited SSH keys, so the generated keypair that is only valid for a limited time period. This is currently set to a maximum of 7 days. For further security, you will also need to <a href="/list/{{ username }}">refresh'</a> your keys at least every 25 hours for the keypair to be valid.</p>
+    <p>Our implementation also considers the use of time-limited SSH keys, so the generated keypair is only valid for a limited time period. This is currently set to a maximum of 7 days. For further security, you will also need to <a href="/list/{{ username }}">refresh</a> your keys at least every 25 hours for the keypair to be valid.</p>
```

Comment:
I suggest to phrase the refresh requirement as

"at least once every 25 hours ..."